site/src/modules/apps/useAppLink.ts

```diff
@@ -50,7 +50,13 @@ export const useAppLink = (
 			// an error message will be displayed.
 			const openAppExternallyFailedTimeout = 500;
 			const openAppExternallyFailed = setTimeout(() => {
-				displayError(`${label} must be installed first.`);
+			        const jetBrainsApps = ["GoLand", "IntelliJ IDEA", "PyCharm", "WebStorm", "CLion"];
+					if (jetBrainsApps.includes(label)) {
+						displayError(`To open ${label}, JetBrains Toolbox must be installed along with the IDE.`);
+					} else {
+						displayError(`${label} must be installed first.`);
+					}
```

Comment:
This should further be filtered when the URI starts with `jetbrains://`. 
Toolbox is launched via this URI.